### ISSUE TITLE
Supporting Apple Silicon

### ISSUE THREAD

**Issue by sabyasachi90** [pszymczyk/embedded-consul#118]
Current version of consul 1.8.x does not work on M1 mac. 
I have created the PR https://github.com/pszymczyk/embedded-consul/pull/117 which upgrades it to 1.11.2 that has support for Apple Silicone

**Comment by oliverlockwood** [comment]
It looks like PR #117 should have closed this.  However, although the README now advises use of version 2.3.1, it appears that version is not available on Maven Central. @pszymczyk was a publish step missed somewhere?

I too am blocked by this issue on my shiny M1 Mac.

**Comment by pszymczyk** [comment]
Yeah, I know that new version is still missing in MC, I need to migrate project to new github pipelines. 

You can always use builder method to override Consul version:
```
    ConsulStarterBuilder withConsulVersion(String consulVersion) {
```

If you'd like you can add manual to readme so other devs will know how to do it :)

**Comment by oliverlockwood** [comment]
Thanks @pszymczyk.  For the reference of others: I was using the JUnit5 `ConsulExtension`.

To specify the Consul version explicitly, I went from (annotation at test class level):
```java
@ExtendWith(ConsulExtension.class)
```

to (within the test class):
```java
    @RegisterExtension
    static ConsulExtension consulExtension = new ConsulExtension(ConsulStarterBuilder.consulStarter()
            .withConsulVersion("1.11.4").build());
```
